Rakefile.dist

```diff
@@ -65,6 +66,14 @@ task :install => :build do |t|
 
   # ssh
   mkdir_p "#{DATAROOTDIR}/foreman/.ssh"
+
+  # cron scripts
+  mkdir_p "#{SYSCONFDIR}/cron.hourly"
+  cp_p "script/passenger-recycler.rb", "#{SYSCONFDIR}/cron.hourly/passenger-recycler"
```

Comment:
Missing some kind of `foreman` prefix, or preferably leave this out and allow packagers to install it in their Foreman-related crontabs, which will be more obvious. This isn't part of Passenger or the default system.